pages/exhibitions/_exhibition/index.vue

```diff
@@ -80,6 +80,14 @@
       mainContent() {
         if (this.page.text === undefined) return;
         return marked(this.page.text);
+      },
+      optimisedImageUrl() {
+        return this.$options.filters.optimisedImageUrl(
+          // use social media image if set in ctf, otherwise use hero image
+          this.page.image ? this.page.image.fields.file.url : this.heroImage.url,
```

Comment:
This needs to be revised to reflect the changes in master from the GraphQL API work whereby the image URL will be in `this.page.image.url`

Reply:
I have update this to reflect the changes from GraphQL. On pages/index.vue I have kept the ternary operator to check if a social share image has been set but checking the exhibition landing pages on contentful none have the option to add a social share image so instead this will just use the hero image by default